tests/integration/009-api-update-loyalty-member-db.spec.ts

```diff
@@ -0,0 +1,194 @@
+		await prisma.truncateTables(['registered_members']);
+		await Promise.all([
+			prisma.insertIntoTable('registered_members', registered_members),
+		]);
+	});
+	test.describe('#PATCH /member/:id', () => {
+		test('Should return success when updating a member with valid information', async ({
+			request,
+		}) => {
+			const res = await request.patch(`api/member/1`, {
+				data: {
+					firstName: 'Updated-FirstName',
+					lastName: 'Updated-LastName',
+					nationality: 'TH',
+					age: 23,
+					phoneNumber: '123456789',
+				},
+			});
+			const body = await res.json();
+
+			await expect(res).toBeOK();
+			expect(body.firstName).toBe('Updated-FirstName');
+			expect(body.lastName).toBe('Updated-LastName');
+			expect(body.nationality).toBe('TH');
+			expect(body.age).toBe(23);
+			expect(body.phoneNumber).toBe('123456789');
+			expect(body.is_active).toBe(true);
+		});
+
+		test('Should return error when updating a member with too long firstName', async ({
+			request,
+		}) => {
+			const res = await request.patch(`api/member/2`, {
+				data: { firstName: 'A'.repeat(51), lastName: 'UpdatedLastName' },
+			});
+			const body = await res.json();
+
+			expect(res.status()).toBe(400);
+			expect(body.result).toBe('FAIL');
+			expect(body.msg).toBe('FirstName or Lastname is longer than 50');
+		});
+
+		test('Should return error when updating a member with too long lastName', async ({
+			request,
+		}) => {
+			const res = await request.patch(`api/member/2`, {
+				data: { firstName: 'UpdatedFirstName', lastName: 'A'.repeat(61) },
+			});
+			const body = await res.json();
+
+			expect(res.status()).toBe(400);
+			expect(body.result).toBe('FAIL');
+			expect(body.msg).toBe('FirstName or Lastname is longer than 50');
+		});
+
+		test('Should return error when updating a member with unsupported nationality', async ({
+			request,
+		}) => {
+			const res = await request.patch(`api/member/3`, {
+				data: { nationality: 'LA' },
+			});
+			const body = await res.json();
+
+			expect(res.status()).toBe(400);
+			expect(body.result).toBe('FAIL');
+			expect(body.msg).toBe('Nationality is not support');
+		});
+
+		test('Should return error when updating a member with age less than 5', async ({
+			request,
+		}) => {
+			const res = await request.patch(`api/member/4`, {
+				data: { age: 4 },
+			});
+			const body = await res.json();
+
+			expect(res.status()).toBe(400);
+			expect(body.result).toBe('FAIL');
+			expect(body.msg).toBe('Age cant be lower than 5');
+		});
+
+		test('Should return error when updating a member with a negative age', async ({
+			request,
+		}) => {
+			const res = await request.patch(`api/member/5`, {
+				data: { age: -1 },
+			});
+			const body = await res.json();
+
+			expect(res.status()).toBe(400);
+			expect(body.result).toBe('FAIL');
+			expect(body.msg).toBe('Age cant be lower than 5');
+		});
+
+		test('Should return error when updating a member with the same phone number', async ({
```

Comment:
This case will be failed, Member id of 1 will have phone number updated successfully because the very first test  `Should return success when updating a member with valid information` has already update phone number of the same member id